lib/contact_list_page.dart

```diff
@@ -1,8 +1,11 @@
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
-
```

Comment:
This undoes the formatting Android Studio automatically gives the imports, and https://dart.dev/guides/language/effective-dart/style#ordering seems to agree that the AS formatting is blessed. Strange that the fmt check doesn't catch that.

Reply:
Maybe it's a separate lint check that we must enable: https://dart.dev/tools/linter-rules#directives_ordering